pkg/core/interop/context.go

```diff
@@ -368,3 +368,15 @@ func (ic *Context) GetBlock(hash util.Uint256) (*block.Block, error) {
 	}
 	return block, nil
 }
+
+// IsHardforkEnabled tells whether specified hardfork enabled at the current context height.
+func (ic *Context) IsHardforkEnabled(hf config.Hardfork) bool {
+	cfg := ic.Chain.GetConfig()
+	hfStr := hf.String()
+	for hf, height := range cfg.Hardforks {
```

Comment:
OK until we have more than 10 of them.

Reply:
Don't know why I wrote this code... We can easily take element from map without this `range`, fixed.